Language.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Language.h";
+
+std::map<std::string, std::string> langfile;
+
+void loadLanguageFile(std::string filename) {
+    std::ifstream file(filename);
+    std::string line;
+    while (std::getline(file, line)) {
+        auto delimiter = line.find("=");
+        if (delimiter != std::string::npos) {
+            auto key = line.substr(0, delimiter);
+            auto value = line.substr(delimiter + 1);
+            langfile[key] = value;
+        }
+    }
+}
+
+std::string localizedString(std::string key) {
+    return langfile[key];

```

Comment:
There is a risk - when we translate to a new language, we may miss some strings. Or add in English and forget to translate. This reflects in a run-time error.
Can you think of a separate test, which I can run to catch this problem, without having to actually run the healthchecker program?